selfdrive/selfdrived/tests/test_alerts.py

```diff
@@ -111,7 +111,7 @@ def test_offroad_alerts(self):
       alert = copy.copy(self.offroad_alerts[a])
       set_offroad_alert(a, True)
       alert['extra'] = ''
-      assert json.dumps(alert) == params.get(a, encoding='utf8')
+      assert alert == params.get(a, encoding='utf8')
```

Comment:
do we really need to specify the encoding? do we ever need anything that's not utf8?

Reply:
Yeah, `CarParams` for example

Reply:
binary type?

Reply:
or even better... capnp type?